app/services/import/legacy/fetch-licence-holders.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+/**
+ * Fetches the Licence Holder data from the import.NALD_PARTIES table for the licence ref
+ * @module FetchLicenceHolderService
+ */
+
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Fetches the licence holder data from the import.NALD_PARTIES table for the licence ref
+ *
+ * The licence holder is determined from the import.NALD_ABS_LIC_VERSIONS table. We get the earliest start date
+ *
+ * @param {string} regionCode - The NALD region code
+ * @param {string} licenceId - The NALD licence ID
+ *
+ * @returns {Promise<ImportLegacyLicenceHolderType[]>}
+ */
+async function go (regionCode, licenceId) {
+  const query = _query()
+
+  const { rows } = await db.raw(query, [regionCode, licenceId])
+
+  return rows
+}
+
+function _query () {
+  return `
+    SELECT DISTINCT ON (np."ID")
```

Comment:
This has been done separately to the previous contact work. 

Maybe it should use the previous contact data. But this functionality encompasses a licence holder (for the company contacts table) as far as i understand it.

So highlighting this is a different thing seemed right ?  

We could merge the two as we really only want to add the contact if we are actually going to link it to a company. ( I will investigate if there will be any edge cases to this understanding)